src/geode/georepresentation/msh_input.cpp

```diff
@@ -78,29 +81,28 @@ namespace std
         std::size_t operator()( const GmshElementID& gmsh_id ) const
         {
             return std::hash< std::string >()( gmsh_id.type.get() )
-                   ^ std::hash< std::string >()( std::to_string( gmsh_id.id ) );
+                   ^ std::hash< geode::index_t >()( gmsh_id.id );
         }
     };
 } // namespace std
 
 namespace
 {
     static constexpr geode::index_t OFFSET_START{ 1 };
+    static std::once_flag once_flag;
```

Comment:
Yes, could also be a member of MSHInputImpl. Not sure if there is a best option.

Reply:
Not if you leave the static keyword